File: python/basic_grammar/operation/assignment_operator/remainder.py
def remainder_assignment_operator():
    a = 51
    a %= 7
    return a


def decimal_remainder_assignment_operator():
    a = 51
    a %= 3.5
    return a


def negative_remainder_assignment_operator():
    a = 51
    a %= -7
    return a

print(remainder_assignment_operator())
print(decimal_remainder_assignment_operator())
print(negative_remainder_assignment_operator())

File: typescript/basic_grammar/operation/assignment_operator/remainder.ts
export const remainderAssignmentOperator = () => {
  let a = 51
  a %= 7
  return a
}

export const decimalRemainderAssignmentOperator = () => {
  let a = 51
  a %= 3.5
  return a
}

export const negativeRemainderAssignmentOperator = () => {
  let a = 51
  a %= -7
  return a
}

console.log(remainderAssignmentOperator())
console.log(decimalRemainderAssignmentOperator())
console.log(negativeRemainderAssignmentOperator())
